finvizfinance/screener/overview.py

```diff
@@ -24,16 +26,41 @@ class Overview:
 
     v_page = 111
 
-    def __init__(self):
+    def __init__(self, username=None, password=None):
         """initiate module"""
+        is_elite = False
+        if username is not None and password is not None:
+            try:
+                auth_cookie = self._get_auth_header(username, password)
+                if auth_cookie is not None:
+                    headers["Cookie"] = auth_cookie
+                    is_elite = True
+            except Exception as e:
+                print(e)
+
         self.BASE_URL = (
-            "https://finviz.com/screener.ashx?v={v_page}{signal}{filter}&ft=4{ticker}"
+            ("https://finviz.com/screener.ashx?v={v_page}{signal}{filter}&ft=4{ticker}")
+            if not is_elite
+            else (
+                "https://elite.finviz.com/screener.ashx?v={v_page}{signal}{filter}&ft=4{ticker}"
+            )
         )
+        print(self.BASE_URL)
         self.url = self.BASE_URL.format(
             v_page=self.v_page, signal="", filter="", ticker=""
         )
         self._load_setting()
 
+    def _get_auth_header(self, email, password):
```

Comment:
I think this method should be moved to util. If needs login, the user can call the function.

Reply:
Please add a description to the method.

Reply:
Done